homework-submission/react-hyf/server/server.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const app = express();
+const path = require('path');
+
+const port = 4000;
+
+app.use(express.static(path.resolve(__dirname, '../react-hyf-client', 'build')));
+
+app.get('/index', function(req, res) {
+  res.sendFile(path.join(__dirname, '../react-hyf-client', 'build', 'index.html'));
```

Comment:
No need for this `get` method since we have the statics doing the exact same thing as here :)